librecad/src/lib/information/rs_information.cpp

```diff
@@ -143,6 +144,31 @@ bool RS_Information::isTrimmable(RS_Entity* e1, RS_Entity* e2) {
     return false;
 }
 
+RS_Entity* RS_Information::getRadiusArc(RS_Entity* e1, RS_Entity* e2) {
+	if (!e1 || !e2)
+		return nullptr;
+	if (!e1->getParent() || !e2->getParent())
+		return nullptr;
+	if (e1->getParent()->rtti()==RS2::EntityPolyline &&
+		e2->getParent()->rtti()==RS2::EntityPolyline &&
+		e1->getParent()==e2->getParent()) {
+		RS_Polyline* poly = (RS_Polyline*)e1->getParent();
+		int idx1 = poly->findEntity(e1);
+		int idx2 = poly->findEntity(e2);
+		unsigned dist = abs(idx1 - idx2);
+		int mid = -1;
+		if (dist == 2)
+			mid = idx1 + (idx2 - idx1) / 2;
+		else if (poly->isClosed() && dist == poly->count() - 2U)
+			mid = (std::max(idx1, idx2) + 1) % poly->count();
```

Comment:
Right, there are two solutions there, and my code always picks up a specific one, without allowing the user to pick the one they want.

Also, it's probably not a good UI anyway, because it would be quicker to just point to an arc directly instead of first and second entity. I can reimplement it that way if you prefer it. So, the fillet command would create a fillet if used on two lines, or modify/delete the fillet if used on an arc. What do you think?